elearning-frontend/src/pages/Dashboard.js

```diff
@@ -17,19 +15,19 @@ const Dashboard = () => {
     const [loading, setLoading] = useState(true)
 
     useEffect(() => {
-        apiClient({
-            method: "get",
-            url: "/api/user"
-        }).then(response => {
-            setUser({
-                ...user,
-                ...response.data,
-                profilepic: (response.data.avatar !== null && response.data.avatar !== "") ? 'http://127.0.0.1:8000/uploads/avatar/' + response.data.avatar : defaultUserPic,
+        const fetchUser = async() => {
+            const data = await apiClient({
+                method: "get",
+                url: "/api/user"
+            }).then(response => {
+                setUser({...response.data})
```

Comment:
No need to destruct if only there is no additional changes.
You can do setUser(response.data)